### PR TITLE
#170542128 view deactivated accommodation

### PR DESCRIPTION
**Pull request by ngireric123**
#### What does this PR do?
- Get deactivated accommodation
#### Description of Task to be completed?
- All  travel admin should get to get all deactivated accommodation
#### How should this be manually tested?
- Clone this branch
- run npm i
- run npm run dev   
- in your browser navigate to localhost:<:port>/deactivated-accommodations

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
- #170542128
#### Screenshots (if appropriate)
image

#### Questions:
N/A

### CODE REVIEW

**Review by sabin18 — APPROVED** [review]
good job @ngireric123 .

**Review by alainmateso — CHANGES_REQUESTED** [review]
Good job @ngireric123, however, ensure that a supplier cannot  view deactivated accommodations as well as see the that 'deactivated accommodations' button because if a supplier clicks it, the app breaks. Also, make sure the buttons align well with the accommodations cards on smaller devices.

**Review by alainmateso — APPROVED** [review]

File: src/components/pages/DeactivatedAccommodation.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  endSearch = () => {
+    this.setState({
+      isSearching: false,
+    });
+    this.componentDidMount();
+  }
+
+  componentDidMount = async () => {
+    this.setState({ isLoading: true });
+    const userInfo = isAuthenticated();
```
Comment (nakiwuge) [inline src/components/pages/DeactivatedAccommodation.js]: Put the const above

File: src/components/pages/DeactivatedAccommodation.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+export class DeactivatedAccommodation extends React.Component {
+  state = {
+    isLoading: false,
+    isSearching: false,
+    userId: null,
+    showSearch: false,
+  };
+
+  showSearch = () => {
```
Comment (nakiwuge) [inline src/components/pages/DeactivatedAccommodation.js:29]: Why are you searching? Was this part of your task?

File: src/components/pages/DeactivatedAccommodation.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    const userInfo = isAuthenticated();
+
+    await this.props.getDeactivatedAccommodation();
+    this.setState({ isLoading: false });
+    await this.setState({
+      userId: userInfo.payload.id,
+    });
+  }
+
+  handleLike = async (slug, action) => {
```
Comment (nakiwuge) [inline src/components/pages/DeactivatedAccommodation.js]: Why are you liking? If you are getting the total number of likes please give your method a better name. something like `getLikes`

File: src/components/pages/DeactivatedAccommodation.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    const { likeUnlikeAccommodation, getDeactivatedAccommodation } = this.props;
+    await likeUnlikeAccommodation(slug, action);
+    await getDeactivatedAccommodation();
+  }
+
+  handleDislike = async (slug, action) => {
```
Comment (nakiwuge) [inline src/components/pages/DeactivatedAccommodation.js]: Same here please give your method a better name. something like `getDislikes`

File: src/components/pages/DeactivatedAccommodation.js [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+    const { likeUnlikeAccommodation, getDeactivatedAccommodation } = this.props;
+    await likeUnlikeAccommodation(slug, action);
+    await getDeactivatedAccommodation();
+  }
+
+  renderAcommodation() {
+    const {
+      accommodationDeactivatedData, searchResults, searchError, showAlert,
+    } = this.props;
+    console.log('acc', accommodationDeactivatedData);
```
Comment (nakiwuge) [inline src/components/pages/DeactivatedAccommodation.js]: Remove console.log

File: src/components/pages/DeactivatedAccommodation.js [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+    let displayItems = {};
```
Comment (nakiwuge) [inline src/components/pages/DeactivatedAccommodation.js:71]: Add a new line

File: src/components/pages/DeactivatedAccommodation.js [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+    }
+    return (
+      <div className="d-flex justify-content-center">
+        <i className="fas fa-spinner fa-pulse loader-big" />
+      </div>
+    );
+  }
+
+  render() {
+    const { isLoading, showSearch } = this.state;
```
Comment (nakiwuge) [inline src/components/pages/DeactivatedAccommodation.js:96]: Add anew line

File: src/components/pages/accommodations/DeactivatedListItem.js [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+import ThumbDownAltIcon from '@material-ui/icons/ThumbDownAlt';
+import ThumbDownOutlinedIcon from '@material-ui/icons/ThumbDownOutlined';
+import ThumbUpAltIcon from '@material-ui/icons/ThumbUpAlt';
+import ThumbUpOutlinedIcon from '@material-ui/icons/ThumbUpOutlined';
+import {
+  LocationOn, StarBorderOutlined, Star,
+} from '@material-ui/icons';
+import BookMark from './BookMark';
+
+
```
Comment (nakiwuge) [inline src/components/pages/accommodations/DeactivatedListItem.js]: Remove this line

**Review by nakiwuge — CHANGES_REQUESTED** [review]
@ngireric123 good job. kindly work on the feedback below.
- Ensure that a user cannot search for an accommodation that is not deactivated on when on the `deactivated accommodations` page.
- Ensure that a user cannot search for an accommodation that is deactivated on when on the `all accommodations` page.
- Give the deactivated accommodations page a heading

**Review by nakiwuge — APPROVED** [review]

**Pull request merged**